lib/vector-buffers/src/encoding.rs

```diff
@@ -177,14 +180,14 @@ impl<T: FixedEncodable> Encodable for T {
     }
 
     fn encode<B: BufMut>(self, buffer: &mut B) -> Result<(), Self::EncodeError> {
-        FixedEncodable::encode(self, buffer)
+        self.encode(buffer)
     }
 
     fn encoded_size(&self) -> Option<usize> {
-        FixedEncodable::encoded_size(self)
+        self.encoded_size()
     }
```

Comment:
These look a little recursive the way they're written. Could we maybe disambiguate somehow?

Reply:
Yeah, I don't know why I made that change.  Gonna revert it.